tests/data/small_sky_object_catalog/properties

```diff
@@ -0,0 +1,6 @@
+#utf-8
```

Comment:
Is this correct? I think I've seen `#HATS catalog` in previous properties files instead.

Reply:
Hah! I just re-ran the `generate_data`, and these were the only two files that changed. Nice catch!